scripts/legacy-kvm-ext4-boot.yaml

```diff
@@ -0,0 +1,47 @@
+#clear-linux-config
+
+# switch between aliases if you want to install to an actuall block device
```

Comment:
S/actuall/actual/g

Reply:
Also capitalize switch and end with period to match the typo pull request.

Reply:
Done. Thanks.